### PR TITLE
Initial QA labels function

### PR DESCRIPTION
**Pull request by ayush1999**
The function `get_qa_tracking_labels()` reads data from the json file and returns a Dict containing id to True/False label. Currently, the label depends on whether the labels `qawanted` or `qe-verify` are present in the data.

Initial changes for #21

### CODE REVIEW

File: bugbug/labels.py [hunk 1]
```diff
@@ -33,6 +33,25 @@ def get_tracking_labels():
...
+    classes = {}
+
+    for bug_data in bugzilla.get_bugs():
+        bug_id = int(bug_data['id'])
+
+        for entry in bug_data['history']:
+            for change in entry['changes']:
+                if change['added'].startswith('qawanted'):
+                    classes[bug_id] = True
+                elif "flags" in entry.keys():
```
Comment (marco-c) [inline bugbug/labels.py]: Always use single quotes for strings instead of double quotes

File: bugbug/labels.py [hunk 2]
```diff
@@ -33,6 +33,25 @@ def get_tracking_labels():
 
     return classes
 
+def get_qa_tracking_labels():
```
Comment (marco-c) [inline bugbug/labels.py]: Let's call this `get_qa_needed_labels()`

File: .gitignore [hunk 3]
```diff
@@ -2,4 +2,4 @@
 *.model
 
 data/bugs.json
-data/commits.json
+data/commits.json
```
Comment (marco-c) [inline .gitignore]: Please remove this unrelated change from the PR

File: bugbug/labels.py [hunk 4]
```diff
@@ -33,6 +33,25 @@ def get_tracking_labels():
...
+    classes = {}
+
+    for bug_data in bugzilla.get_bugs():
+        bug_id = int(bug_data['id'])
+
+        for entry in bug_data['history']:
+            for change in entry['changes']:
+                if change['added'].startswith('qawanted'):
+                    classes[bug_id] = True
+                elif 'flags' in entry.keys():
```
Comment (marco-c) [inline bugbug/labels.py]: No need to do `entry.keys()`, you can just do `'flags' in entry`.

**Comment by marco-c** [comment]
Almost perfect, just need to do a small change I suggested and you should fix the flake8 issues (you can see them on Travis, or you can run flake8 locally).

**Comment by ayush1999** [comment]
@marco-c Done!
